app/Console/Commands/AdsharesCrawlCommand.php

```diff
@@ -78,7 +86,7 @@ protected function readBroadcasts(Esc $esc)
                 $this->info("Found $host -> {$log['address']}");
                 // TODO: extract algo
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)